packages/client/src/views/SearchResult/SearchResult.tsx

```diff
@@ -281,6 +283,18 @@ class SearchResultView extends React.Component<
           params.get('searchText'),
           params.get('searchType')
         ]
+        const isDeclarationReviewableByRegistrar =
+          declarationIsValidated ||
+          declarationIsCorrectionRequested ||
+          isDeclared ||
+          declarationIsInProgress
+
+        const isDeclarationReviewableByRegAgent =
+          isDeclared || declarationIsInProgress || declarationIsRejected
```

Comment:
not too sure about the rejected one being reviewable by RA :sweat_smile:

Reply:
Thanks @Zangetsu101 , as I was also not completely sure, I verified with Syeda that a RA can review rejected declarations. In addition to that I also I added to registrar review condition here in this commit https://github.com/opencrvs/opencrvs-core/pull/7226/commits/75484b87b550ffefff5a84b461c467e02c028ad1

Reply:
This is so much cleaner now @tahmidrahman-dsi ! Good work.